wrangler-transform/pom.xml

```diff
@@ -99,7 +99,7 @@
               <Embed-Transitive>true</Embed-Transitive>
               <Embed-Directory>lib</Embed-Directory>
               <!--Only @Plugin classes in the export packages will be included as plugin-->
-              <_exportcontents>co.cask.wrangler.api.*</_exportcontents>
```

Comment:
The Wrangler.java transform class was no longer being included in CDAP as a Plugin, since its not in the co.cask.wrangler.api.* packages.